app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue

```diff
@@ -79,6 +79,7 @@
                   v-model="message"
                   class="message-editor"
                   :class="{ editor_warning: $v.message.$error }"
+                  :enable-variables="true"
```

Comment:
Are there flags at the global level that can be set instead of hard-coding this? Like `$featureFlags.variables`? 

CC: @fayazara